src/modules/game.module.ts

```diff
@@ -13,23 +13,29 @@ import EventHandlers from '../Hangman/Domain/EventHandlers';
 import { GamesResolver } from '../resolvers/game.resolver';
 import { Game as GameProjection } from '../Hangman/ReadModels/game.entity';
 import { EventStoreInstanciators } from '../event-store';
+import { MongoStore } from '../mongo/mongo-eventstore-adapter';
 
 @Module({
   imports: [
     CqrsModule,
-    EventStoreModule.registerFeature({
-      featureStreamName: '$ce-game',
+    EventStoreModule.registerFeatureAsync({
       type: 'event-store',
-      subscriptions: [
-        {
-          type: EventStoreSubscriptionType.CatchUp, // research various types
-          stream: '$ce-game',
-          resolveLinkTos: true, // Default is true (Optional)
-          lastCheckpoint: null,
-          //fetches from the start. in follow-up PR, store the position somewhere, and setup a configservice that can read this position and insert it here
-        },
-      ],
-      eventHandlers: EventStoreInstanciators,
+      useFactory: async () => {
+        const streamName = 'game';
+
+        return {
+          type: 'event-store',
+          featureStreamName: `$ce-${streamName}`,
+          store: MongoStore(streamName),
```

Comment:
here https://github.com/juicycleff/nestjs-event-store#setup-feature-module-1
it looks like `MongoStore` is a class, but it's not instanciated (`new MongoStore(streamName)`)
How would this work to use the static class? Still need to pass parameter streamName?